### DISCUSSION TITLE
Use Tradingview's open source charting

### DISCUSSION THREAD

**Discussion by rdong8** [questdb/questdb#1697] [Ideas]
### Is your feature request related to a problem?

The current charts in the web console are not interactive, and especially given QuestDB's heavy usage with financial data, I think it would be a useful feature to have, so that one could ex. zoom in on a stock's price or something. 

### Describe the solution you'd like.

Tradingview has an [open source](https://github.com/tradingview/lightweight-charts) HTML5 charting library that is really amazing. See the demos [here](https://www.tradingview.com/lightweight-charts/). They are clean and configurable, load fast, can be actively updated, etc.

**Comment by mationai** [comment]
Not sure why this is labeled as "New Feature". It should be more of "Applications" or "Demo". 

But since we're on the topic, for anyone looking, this may save you days - weeks. I am using questdb for stock charting and I've looked at all the open source charting libraries on github and discovered a very capable one, it's https://github.com/liihuu/KLineChart . It's may not be as powerful as tradingview's commercial chart, but it blows the water out of its OSS one (lightweight chart) on most features I care about. I wish I could have taken back the weeks I've spent dealing w/ lightweight chart and other charts I tried. Been using it for over a year, have nothing but good things to say about it.
